api/database/migrations/2020_08_20_213138_add_social_accounts_table.php

```diff
@@ -15,11 +15,11 @@ public function up()
     {
         Schema::create('social_accounts', function (Blueprint $table) {
             $table->unsignedBigInteger('user_id');
-            $table->unsignedBigInteger('provider_id');
-            $table->unsignedTinyInteger('account_id');
+            $table->unsignedBigInteger('provider_id')->nullable();
+            $table->string('account_id');
             $table->longText('token');
-            $table->longText('refresh_token');
-            $table->timestamp('expires_in');
+            $table->longText('refresh_token')->nullable();
+            $table->timestamp('expires_in')->nullable();
```

Comment:
Changing existing migration will not have an effect because it is already applied. Also, the provider id can not be null.